eslint.config.json

```diff
@@ -0,0 +1,41 @@
+{
+  "env": {
```

Comment:
You can delete this file, I made a mistake in the content. `eslintrc` is the correct one :)